examples/bakdata/atm-fraud-detection/pipeline.yaml

```diff
@@ -10,6 +10,7 @@
       jmx:
         enabled: false
     debug: true
+  prefix: ""
```

Comment:
Why do we need this?

Reply:
we don't actually, but we are replacing the global config for a localone right ? so I am only adapting with this global config https://github.com/bakdata/kpops/blob/main/examples/bakdata/atm-fraud-detection/config.yaml#L15

Reply:
what do you think ?

Reply:
Then define it once for kubernetes-app. Ideally we could also set it for PipelineComponent but that does not have a type. Any objections to set one so we can configure all components @raminqaf @disrupted ?

Reply:
I am not against the idea. This would need to be tested for the schema generation though.

Reply:
so the idea would be to move the prefix field to kubernetes app ? I didn't get it

Reply:
Change the defaults.yaml in the examples folder and set the prefix for kubernetes-app

Reply:
like this ?

Reply:
Now you can remove it in this file. For connectors you also need to specify it in the defaults